doc/package-notes.xml

```diff
@@ -814,4 +814,46 @@ citrix_receiver.override {
    </para>
   </section>
  </section>
+ <section xml:id="sec-ibus-typing-booster">
+  <title>ibus-engines.typing-booster</title>
+
+  <para>This package is an ibus-based completion method to speedup typing.</para>
+
+  <section xml:id="sec-ibus-typing-booster-activate">
+   <title>Activate engine</title>
+
+   <para>
+    IBus needs to be configured accordingly to activate <literal>typing-booster</literal>. The configuration
+    depends on the desktop manager in use. For detailed instructions, please refer to the
+    <link xlink:href="https://mike-fabian.github.io/ibus-typing-booster/documentation.html">upstream docs</link>.
+   </para>
+   <para>
+    On NixOS you explicitly need to enable <literal>ibus</literal> with given engines
```

Comment:
Also I would move the “explicitly” after the “need to” or after “with given engines”.